tests/foreman/api/test_hostcollection.py

```diff
@@ -37,6 +48,35 @@ def setUpClass(cls):
         super(HostCollectionTestCase, cls).setUpClass()
         cls.org = entities.Organization().create()
         cls.hosts = [entities.Host(organization=cls.org).create() for _ in range(2)]
+        cls.lce = entities.LifecycleEnvironment(organization=cls.org).create()
+        cls.content_view = entities.ContentView(organization=cls.org).create()
+        cls.content_view.publish()
+        cls.content_view = cls.content_view.read()
+        promote(cls.content_view.version[0], environment_id=cls.lce.id)
+        cls.activation_key = entities.ActivationKey(
+            content_view=cls.content_view, environment=cls.lce, organization=cls.org
+        ).create()
+        setup_org_for_a_rh_repo(
```

Comment:
Do you need that?
As I understand, it synchronizes Red Hat repo and adds relevant subscription to activation key. But you can attach subscription to host without ever synchronizing repository. Should save you some time during execution.

Reply:
Yes, has now been removed